speech/google/cloud/speech/client.py

```diff
@@ -46,26 +46,17 @@ class Client(client_module.Client):
 
     _connection_class = Connection
 
-    def async_recognize(self, content, source_uri, encoding, sample_rate,
+    def async_recognize(self, encoding, sample_rate, content=None,
```

Comment:
@daspecster Maybe add `default_encoding` and `default_sample_rate` as properties of the `Client`, and then use them when constructing the `Sample` if the user doesn't pass in explicit `encoding` or `sample_rate` values?